Error message on install: ./quick.go:16:8: client.QuickCommand undefined (type *todoist.Client has no field or method QuickCommand)

Hi, this looks super cool and I'd like to try it, but I got the above error on installing. I believe the install was correct -- my workspace is ~/.go vs the usual ~/go, but my path is properly modified and other Go stuff works. Unfortunately, I'm just starting with Go and am not able to diagnose the error. Any help or advice you might offer would be much appreciated and diligently acted upon.

If this helps I'm running Ubuntu Gnome 16.04.3, and here is the series of terminal commands and their output. If more data would help, let me know and I'll gladly provide it.

`# ken @ devbox in ~/.go/src [2:10:33] 
$ git clone https://github.com/sachaos/todoist.git
Cloning into 'todoist'...
remote: Counting objects: 782, done.
remote: Compressing objects: 100% (18/18), done.
remote: Total 782 (delta 8), reused 20 (delta 6), pack-reused 757
Receiving objects: 100% (782/782), 134.02 KiB | 0 bytes/s, done.
Resolving deltas: 100% (515/515), done.
Checking connectivity... done.

# ken @ devbox in ~/.go/src [2:10:40] 
$ ls
github.com  hello  todoist

# ken @ devbox in ~/.go/src [2:10:42] 
$ cd todoist 

# ken @ devbox in ~/.go/src/todoist on git:master o [2:10:45] 
$ make install
dep ensure
go get golang.org/x/tools/cmd/goyacc
goyacc -o filter_parser.go filter_parser.y

conflicts: 4 shift/reduce
go install
# todoist
./quick.go:16:8: client.QuickCommand undefined (type *todoist.Client has no field or method QuickCommand)
Makefile:3: recipe for target 'install' failed
make: *** [install] Error 2

# ken @ devbox in ~/.go/src/todoist on git:master x [2:11:37] C:2
$ go install
# todoist
./quick.go:16:8: client.QuickCommand undefined (type *todoist.Client has no field or method QuickCommand)
`

--- a/README.md
+++ b/README.md
@@ -86,6 +86,8 @@ $ brew install todoist
 You need version 1.7 or higher Golang, and [golang/dep: Go dependency management tool](https://github.com/golang/dep).
 
 ```
+$ mkdir -p $GOPATH/src/github.com/sachaos
+$ cd $GOPATH/src/github.com/sachaos
 $ git clone https://github.com/sachaos/todoist.git
 $ cd todoist
 $ make install

